example/fine_advection/advection_package.hpp

```diff
@@ -33,14 +33,23 @@ namespace advection_package {
 using namespace parthenon::package::prelude;
 
 namespace Conserved {
-VARIABLE(advection, scalar);
-VARIABLE(advection, scalar_fine);
-VARIABLE(advection, scalar_fine_restricted);
+VARIABLE(advection, phi);
+VARIABLE(advection, phi_fine);
+VARIABLE(advection, phi_fine_restricted);
+VARIABLE(advection, C);
+VARIABLE(advection, D);
```

Comment:
given these could be anything, I guess C and D are fine names but they seem rather non standard. I would prefer A (or E) and B or psi and pi (to go with phi)

Reply:
I explicitly tried to avoid $\vec{A}$, $\vec{B}$, and $\vec{E}$.

Reply:
I figured... Well my preference is for psi and pi but I suppose it doesn't matter much

Reply:
I think I am going to leave $\vec{C}$ and $\vec{D}$. $\psi$ and $\pi$ seem scalary to me, and more importantly keeping it the same requires no search and replace...